### ISSUE TITLE
Link / Autolink HTML Integration

### ISSUE THREAD

**Issue by daevem** [dvorka/mindforger#909]
Hi there, 
I just want to point out an issue in rendering links inside HTML tags. In fact, if i write something like:
`<p>[firewall](/home/userx/minforger-repository/memory/Security.md#firewall)</p>`
MF will render exaclty the same string and **not** the actual link. 
This wouldn't be too much a problem, since one could simply avoid using links inside HMTL tags, however when Autolink is toggled this happens automatically. Here's an example:

<img width="172" alt="capture2" src="https://user-images.githubusercontent.com/36861140/49869168-de291900-fe0f-11e8-9110-4fbd51d223f1.PNG">
<img width="290" alt="capture1" src="https://user-images.githubusercontent.com/36861140/49869294-4bd54500-fe10-11e8-9bae-7dbdb2f14ef1.PNG">

**Comment by dvorka** [comment]
@daevem you are right - I released autolinking (as preview feature), but probably it was too early. I'm aware of this issue and have new design for autolinking ready. I hope to bring fast and correct autolinking implemenation in the next feature release https://github.com/dvorka/mindforger/milestone/44

Thank you for sending the issue and your interest in MindForger!
